src/api_client.ts

```diff
@@ -0,0 +1,89 @@
+import restClient, {Options, RequestPromise} from 'request-promise';
+import { createGovAuNotifyToken } from './authentication';
+import packageJson from '../package.json';
+
+export default class NotifyClient {
```

Comment:
Can just be named `Client`. Or, since we have another client in src/notification.ts maybe name this one `APIClient`?

Reply:
Renamed to httpClient